docs/compatibility_matrix.md

```diff
@@ -0,0 +1,14 @@
+# Compatibility Matrix
+
+This document shows the compatibility of Scalar DB with supported databases.
+
+## Scalar DB Compatibility
+
+| Scalar DB Version   | Cassandra 3.11.10 | Cassandra 3.0.24    | Cassandra 2.2.19   | Cosmos DB | DynamoDB  | MySQL 8.0.25  | MySQL 2.7.34  | PostgreSQL 13.3    | PostgreSQL 12.7   | PostgreSQL 11.12  |
```

Comment:
MySQL 2.7 sounds very old.
Is it a mistake? and 5.7?

Reply:
Oh, sorry!
It's my mistake, 5.7 is correct.